packages/x-components/src/components/result/base-result-image.vue

```diff
@@ -171,4 +240,12 @@ displayed while the real one is loaded.
   </template>
 </BaseResultImage>
 ```
+
+### Showing the next image on hover
+
+If a result has multiple images, it can show the next one on hover.
```

Comment:
Does it make sense to include here also how to set an animation? So we document the two new props in the same example 🤔

Reply:
Sure!